src/client/mds_client.h

```diff
@@ -39,25 +38,19 @@
 #include "src/client/client_metric.h"
 #include "src/client/mds_client_base.h"
 #include "src/client/metacache_struct.h"
-#include "src/common/concurrent/concurrent.h"
-#include "src/common/concurrent/rw_lock.h"
 
 namespace curve {
 namespace client {
 
-using curve::common::RWLock;
-using curve::common::ReadLockGuard;
-using curve::common::Authenticator;
-
-class MetaCache;
 struct LeaseRefreshResult;
 
 // MDSClient是client与MDS通信的唯一窗口
-class MDSClient {
+class MDSClient : public MDSClientBase,
```

Comment:
Why dose ‘MDSClient’ change to inherit from 'MDSClientBase' ?

Reply:
After removing the option from MDSClientBase, it becomes an empty class. So if we define a member variable of class MDSClientBase, it at least occupies 1-byte memory. To avoid taking up extra space, inherit it is a good solution. Also, make MDSClient inherits from MDSClientBase has no bad effect.